src/router/routes.js

```diff
@@ -98,10 +97,7 @@ const routes = [
         path: 'errata',
         component: () => import('pages/ErrataFeed.vue'),
         beforeEnter(to, from, next) {
-          store
-            .dispatch('platforms/loadPlatformList')
-            .then(next())
-            .catch(next())
+          store.dispatch('platforms/loadPlatformList').then(next())
```

Comment:
I think I found the cause. `dispatch()` returns a Promise, so `then(next())` does not wait for `loadPlatformList` to complete.
```suggestion
          store.dispatch('platforms/loadPlatformList').then(() => next())
```

Reply:
ohh, I see, can you try now?

Reply:
Yes, this fixes my issue. Thanks a lot!
(There are other `then(next())` in route.js, which might not be intentional either.)